sites/frontend/components/Header/Nav/SubNav/SubNavList/SubNavListItem/SubNavButton.js

```diff
@@ -59,7 +61,13 @@ const SubNavButton = styled('button')(
 );
 SubNavButton.displayName = 'SubNavButton';
 
-export default ({ pillar, showSecondaryNav, toggleSecondaryNav }) => {
+type Props = {
+    pillar: PillarType,
+    showSecondaryNav: boolean,
+    toggleSecondaryNav: Function,
```

Comment:
Maybe make this more explicit

```
toggleSecondaryNav: () => void
```

Flow docs [warn against](https://flow.org/en/docs/types/functions/#toc-function-type) using the `Function` type as it almost completely opts out of the type checker

Reply:
i just read that actually - nice one